MANIFEST

```diff
@@ -5479,6 +5479,7 @@ Porting/sync-with-cpan		Sync with CPAN
 Porting/timecheck.c		Test program for the 2038 fix
 Porting/timecheck2.c		Test program for the 2038 fix
 Porting/todo.pod		Perl things to do
+Porting/updateAUTHORS.pl	Tool to automatically update AUTHORS and .mailmap from git log data
```

Comment:
do we need to ship this tool? 
Would it be used outside the GitHub repo?
note that `.mailmap` is not even listed in the MANIFEST

Reply:
"do we need to ship this tool"

Probably not, but that applies to a number of things in Porting and IMO is orthogonal to this PR.

"Would it be used outside the GitHub repo?"
No, but this point would also apply to Porting/checkAUTHORS.pl as well wouldn't it? Nit: its a git repo, github just hosts the repo. :-)

Maybe we should add a new directory that is excluded from tarballs. But I can also imagine that is a can of worms. It certainly doesn't cause any *harm* shipping them.

Also tools in Porting aren't installed or anything, but they are included in the build tarball, so I think its like shipping any of our build tools. We have many tools in there that would be useless without git, eg, Porting/bisect.

"note that .mailmap is not even listed in the MANIFEST"
That suggests there might be a bug in the MANIFEST tests and an oversight I guess. But also I think something that should be addressed in a separate issue/PR.

Reply:
I created https://github.com/Perl/perl5/issues/19512 and https://github.com/Perl/perl5/issues/19511 to address these points.

Reply:
thanks, this makes sense to extend the discussion to other files too